apps/nextjs-website/src/lib/strapi/tutorial.ts

```diff
@@ -0,0 +1,61 @@
+import * as t from 'io-ts/lib';
+import * as tt from 'io-ts-types';
+import * as qs from 'qs';
+import { fetchFromStrapi } from './fetchFromStrapi';
+import { DateFromISOString } from 'io-ts-types/lib/DateFromISOString';
+import { MediaCodec } from './codecs/MediaCodec';
+import { NullToUndefinedCodec } from './codecs/NullToUndefinedCodec';
+import { BlocksContentCodec } from './codecs/BlocksContentCodec';
+import { PaginationCodec } from './codecs/PaginationCodec';
+import { ProductCodec } from './codecs/ProductCodec';
+import { RelatedLinksCodec } from './codecs/RelatedLinksCodec';
+
+const BannerLinkCodec = t.strict({
+  id: t.number,
+  title: t.union([NullToUndefinedCodec, t.string]),
+  body: t.union([NullToUndefinedCodec, BlocksContentCodec]),
```

Comment:
What about `icon` and `theme`?

```
type BannerLinkProps = {
  id: number;
  theme?: 'dark' | 'light';
  title?: string;
  body?: BlocksContent;
  icon: {
    readonly name: string;
    readonly alternativeText: string | null;
    readonly width: number;
    readonly height: number;
    readonly ext: string;
    readonly mime: string;
    readonly url: string;
  };
};
```

Reply:
I know some attributes are added a next PR and about other missing attribute i thinnk is better to manage the refactor in a separeted task so we can align Strapi type and all FE occurencies

Reply:
https://pagopa.atlassian.net/browse/DEV-1725